_posts/2018-09-17-not-your-kind-of-automation.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title:  "Not your kind of automation"
+author: "Nuno Santos"
+date:   2018-09-17
+thumbnail: /img/posts/not-your-kind-of-automation/manual-vs-automated.png
+categories: [engineering-quality]
+tags: [automation,teams]
+permalink: /not-your-kind-of-automation/
+---
+
+<center>
+	<img alt="Manual vs Automated" src="/img/posts/not-your-kind-of-automation/manual-vs-automated.png" style="width: 90%; margin-bottom: 10px;" />
+</center>
+
+## Introduction
+
+Oh no, another post about automation? Well, kinda.
+
+When we talk about automation on our industry we first think about deploys, continuous integration, delivery and so on. But, I won’t be talking about that today, it’s, let’s say, another level, important part of it. How many times you’ve done a very boring repetitive task that takes your attention for quite some time and you get frustrated by not having time for doing the cool stuff?
+
+…
+
+Yeah, that’s right, I know the feeling.
+
+## Background
+
+Inside Jumia, everyone knew that I had a huge set of scripts that did a lot of different things. Although that is good, the scripts were for my usage “only”. So the idea of sharing them was born. To make it happen, we thought of implementing a basic but powerful UI, which allows, execution of such scripts easily and be very user friendly. And so, we created JDash.
+
+(At the end I share JDash stack for the curious ones.)
+
+## Why does it matter
+
+You can improve everyone’s work by doing so, from Developers to Top Management. The sky’s the limit. Also, you motivate your team by doing something different, you empower them, you make them more noticeable, and everyone can be involved in the tool / product evolution. So, how do I say this, our industry is full of talent, but, we still do not automate our most common tasks. And, honestly, we all are the one’s to blame. We do tech, so why don’t we take the time to improve our workflow by creating something that eases your and everyone’s life?
```

Comment:
> be involved in the tool / product evolution. 

e involved in the **tool/product** evolution. 

> all are the one’s to blame

all are the **ones** to blame

Reply:
Done!